packages/flask/src/UICarouselView/CarouselView/CarouselContainer/index.native.tsx

```diff
@@ -0,0 +1,106 @@
+import React from "react"
+import { throttle } from "lodash";
```

Comment:
```suggestion
import throttle from "lodash/lodash";
```

Otherwise it will import the whole `lodash` which we don't need to have in the bundle. We only need the `throttle` module here.